src/main.lua

```diff
@@ -812,15 +806,33 @@ addEventingHandler(ActionMap.BuyRecord, utils.hasMatchingTag("Action", ActionMap
 	Send(msg, {
 		Target = msg.From,
 		Tags = { Action = ActionMap.BuyRecord .. "-Notice", Name = name },
-		Data = json.encode(fundFrom and result or {
```

Comment:
What was the motivation for this conditional? 🤔

Reply:
not sure, but this was resulting very verbose payload when fundFrom was true. e.g 
```
 baseRegistrationFee: number,
    df: table,
    fundingPlan: table,
    fundingResult: table,
    protocolBalance: number,
    record: Record,
    remainingBalance: number,
    returnedName: ReturnedNameBuyRecordResult|nil,
    totalRegistrationFee: number,
```

Reply:
Much of that is useful return data aside from maybe protocolBalance.